scripts/image

```diff
@@ -259,6 +259,12 @@ if [ "$1" = "release" -o "$1" = "mkimage" -o "$1" = "amlpkg" -o "$1" = "noobs" ]
 
   RELEASE_DIR="target/$IMAGE_NAME"
 
+  if [ -n "$DEVICE" -a -d "$PROJECT_DIR/$PROJECT/devices/$DEVICE/install" ]; then
+    INSTALL_SRC_DIR="$PROJECT_DIR/$PROJECT/devices/$DEVICE/install"
+  else
+    INSTALL_SRC_DIR="$PROJECT_DIR/$PROJECT/install"
+  fi
+
```

Comment:
I think the above code should be in a commit of it's own ie scripts/image: INSTALL_SRC_DIR add device support  …

Reply:
I don't understand what you mean here. I need to pass `$INSTALL_SRC_DIR` to `mkimage`, that's why I moved defining it earlier.

Reply:
Sorry I'm thinking of the per device filesystem